Description: Fix interpreter lines in examples to match Debian locations
Forwarded: not-needed
Author: Rowan Thorpe <rowan@rowanthorpe.com>
Last-Update: 2013-09-26
--- a/scripts/ipTopTalkers
+++ b/scripts/ipTopTalkers
@@ -1,4 +1,4 @@
-#!/bin/awk -f
+#!/usr/bin/awk -f
 #
 # Copyright (c) 2001 InMon Corp. Licensed under the terms of the InMon sFlow licence:
 # http://www.inmon.com/technology/sflowlicense.txt
--- a/scripts/ipTrafficMatrix
+++ b/scripts/ipTrafficMatrix
@@ -1,4 +1,4 @@
-#!/bin/awk -f
+#!/usr/bin/awk -f
 #
 # Copyright (c) 2001 InMon Corp. Licensed under the terms of the InMon sFlow licence:
 # http://www.inmon.com/technology/sflowlicense.txt
--- a/scripts/sflowRRDChart.cgi
+++ b/scripts/sflowRRDChart.cgi
@@ -1,4 +1,4 @@
-#! /usr/local/rrdtool-1.0.33/bin/rrdcgi
+#!/usr/bin/rrdcgi
 # Copyright (c) 2001 InMon Corp. Licensed under the terms of the InMon sFlow licence:
 # http://www.inmon.com/technology/sflowlicense.txt
 <HTML>
--- a/scripts/sflowRRDLoad
+++ b/scripts/sflowRRDLoad
@@ -1,4 +1,4 @@
-#! /usr/bin/perl
+#!/usr/bin/perl
 # Copyright (c) 2001 InMon Corp. Licensed under the terms of the InMon sFlow licence:
 # http://www.inmon.com/technology/sflowlicense.txt
 
